### PR TITLE
Switch `attributions2json` over to use GitHub's public API instead of SVN

### LINKED ISSUE #3: SVN command times out when transforming source file into json
Source file:
```
https://github.com/fastlane/fastlane
```

Output:
```
$ attributions2json.py customAttributions.source customAttributions.json                                                                                                                         

svn: E175002: Unexpected HTTP status 504 'Gateway Timeout' on '/fastlane/fastlane/trunk'
fastlane: license file not found
```

### PR DESCRIPTION
**Pull request by Johennes**
This avoids running into timeouts when listing contents of large repositories
over SVN.

Further minor changes included:

- Switch method and variable names to snake case in accordance with PEP 8
- If no file "LICENSE*" is found, fall back to "COPYING*"
- Added note about rate limits and access token to README, also cleaned up formatting a bit
- Added custom attributions example

Fixes #3

### CODE REVIEW

**Comment by Johennes** [comment]
@trevor-e / @juraj-blahunka: adding both of you for review. One is probably enough though so whoever has cycles for it first.

File: attributions2json.py [hunk 1]
```diff
@@ -25,39 +25,43 @@ def exportJSON(output, *attributions):
...
 
 
-def getLicenseTextFrom(gitRepo):
-    filename = getLicenseFileNameFrom(gitRepo)
-    if filename:
-        command = '{} {}/{}'.format('svn cat', gitRepo, filename)
-        return spawnProcessWith(command)
+def send_content_request(repo, path):
+    request = urllib2.Request('https://api.github.com/repos/%s/contents/%s' % (get_owner_name(repo), path.lstrip('/')),
+        headers = { 'Authorization': 'token %s' % os.environ['GITHUB_ACCESS_TOKEN'] })
```
Comment (trevor-e) [inline attributions2json.py]: Maybe print a warning if `GITHUB_ACCESS_TOKEN` is unset?

File: attributions2json.py [hunk 2]
```diff
@@ -25,39 +25,43 @@ def exportJSON(output, *attributions):
...
+    data = send_content_request(repo, '/')
+    item = next((item for item in data if item['path'].startswith('LICENSE')), None)
+    if not item:
+        item = next((item for item in data if item['path'].startswith('COPYING')), None)
+    if item:
+        return item['path']
 
 
-def validateFile(file, type):
+def fetch_license_text(repo):
```
Comment (trevor-e) [inline attributions2json.py]: µ-nit: `fetch` when other functions are `get`

**Review by trevor-e — APPROVED** [review]
Looks good! 👍

**Pull request merged**